src/tools/clippy/tests/ui/crashes/ice-10645.stderr

```diff
@@ -4,11 +4,14 @@ warning: future cannot be sent between threads safely
 LL | pub async fn bar<'a, T: 'a>(_: T) {}
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ future returned by `bar` is not `Send`
    |
-note: captured value is not `Send`
-  --> tests/ui/crashes/ice-10645.rs:5:29
+note: future is not `Send` as this value is used across an await
+  --> tests/ui/crashes/ice-10645.rs:5:35
    |
 LL | pub async fn bar<'a, T: 'a>(_: T) {}
-   |                             ^ has type `T` which is not `Send`
+   |                                   ^-
+   |                                   |
+   |                                   await occurs here, with `{}` maybe used later
+   |                                   has type `T` which is not `Send`
```

Comment:
The Clippy changes do not look correct. The new `note` now claims, that the body of the function is maybe used later.  You probably have to fix the Clippy code, rather than just update the stderr files.

Reply:
I see. I will work on restoring the identification on the upvar instead of erring on the `UNRESUME` variant.

Reply:
I would like to report that the span is now corrected.

Reply:
Thanks!